corehq/ex-submodules/pillowtop/tasks.py

```diff
@@ -20,6 +20,10 @@ def _is_couch(pillow):
 
     pillow_meta = get_all_pillows_json()
 
+    active_pillows = getattr(settings, 'ACTIVE_PILLOW_NAMES')
```

Comment:
I think this will fail if `ACTIVE_PILLOW_NAMES` isn't set:

```
n [1]: settings = object()

In [2]: active_pillows = getattr(settings, 'ACTIVE_PILLOW_NAMES')
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-2-250fb84f683c> in <module>()
----> 1 active_pillows = getattr(settings, 'ACTIVE_PILLOW_NAMES')

AttributeError: 'object' object has no attribute 'ACTIVE_PILLOW_NAMES'

In [3]: active_pillows = getattr(settings, 'ACTIVE_PILLOW_NAMES', None)

In [4]:
```

`getattr` behaves differently form `dict.get`. `d.get(key)` is the same as `d.get(key, None)`, but `getattr(obj, property)` is analogous to `d[property]`—you have to pass in a default value (`None` here) explicitly if you want not to fail if the property is missing.

Reply:
Thanks Danny, missed that simple thing.